components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -162,6 +202,9 @@ export default class AtMentionProvider extends Provider {
             if (user.username) {
                 return '@' + user.username;
             }
+            if (user.display_name) {
+                return '@' + user.display_name;
+            }
```

Comment:
This would be a `group` right? maybe the code should be updated to be resource agnostic into something like `item` or `userOrGroup`

Reply:
yes, it is on my list of things to refactor here.